doc/architecture/decisions/0036-use-audited-for-budget-revisions.md

```diff
@@ -0,0 +1,53 @@
+# 36. Use Audited for Budget Revisions
+
+Date: 2023-03-15
+
+## Status
+
+Accepted
+
+## Context
+
+Budgets can go through a number of revisions; they can be updated over time as
+more or less funding comes in, or the scope of the activity changes.
```

Comment:
Can "less funding" _come in_? Perhaps "as funding increases or decreases"?

Reply:
I actually stole this from the Zendesk guidance so I'm leaving it as is!